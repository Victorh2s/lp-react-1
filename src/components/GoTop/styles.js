import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${({ theme }) => css`
    position: fixed;
    background: none;
    color: ${theme.colors.colorbuttons};
    border: 0.1rem solid ${theme.colors.colorbuttons};
    border-radius: 5rem;
    display: flex;
    align-items: center;
    justify-content: center;
    width: 4rem;
    height: 4rem;
    bottom: 2rem;
    right: 2rem;
    z-index: 6;
  `}
`;
