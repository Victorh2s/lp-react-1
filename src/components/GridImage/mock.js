export default {
  background: false,
  title: 'My grid',
  description:
    'Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde',
  grid: [
    {
      id: 1,
      url: 'https://source.unsplash.com/random/800x800?r=1',
    },
    {
      id: 2,
      url: 'https://source.unsplash.com/random/800x800?r=2',
    },
    {
      id: 3,
      url: 'https://source.unsplash.com/random/800x800?r=3',
    },
    {
      id: 4,
      url: 'https://source.unsplash.com/random/800x800?r=4',
    },
    {
      id: 5,
      url: 'https://source.unsplash.com/random/800x800?r=5',
    },
    {
      id: 6,
      url: 'https://source.unsplash.com/random/800x800?r=6',
    },
  ],
};
