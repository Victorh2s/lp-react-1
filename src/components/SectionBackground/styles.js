import styled, { css } from 'styled-components';

const backgroundActive = (theme) => css`
  background: ${theme.colors.primaryColor};
  color: ${theme.colors.white};
`;

export const Container = styled.div`
${({ theme, background }) => css`
transition: all 300ms ease-in-out;
background: ${theme.colors.white};
color: ${theme.colors.primaryColor};
${background && backgroundActive(theme)}
min-height: 80vh;
display: flex;
align-items: center;
`}
`;
