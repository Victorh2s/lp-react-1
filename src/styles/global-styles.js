import { createGlobalStyle, css } from 'styled-components';

export const GlobalStyles = createGlobalStyle`
  @import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,600;0,800;1,700&display=swap');
  * {
    margin:0;
    padding:0;
  }
  html {
    font-size: 62.5%;
  }
  body {
    font-size:2.4rem;
    font-family: ${({theme})=> theme.font,family.default};
  }
  h1 {
    font-size: 2.4rem;
    font-family:  ${({theme})=> theme.font,family.secondary};
  }
`;
