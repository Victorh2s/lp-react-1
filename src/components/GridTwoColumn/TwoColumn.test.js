import { GridTwoColumns } from '.';
import { renderTheme } from '../../styles/render-theme';
import mock from './mock';

describe('<GridTwoColumn/>', () => {
  it('should render two column', () => {
    const { container } = renderTheme(<GridTwoColumns {...mock} />);
    expect(container).toMatchSnapshot();
  });
});
