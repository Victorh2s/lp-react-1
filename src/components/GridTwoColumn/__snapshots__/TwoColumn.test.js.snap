// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridTwoColumn/> should render two column 1`] = `
<div>
  <div
    class="sc-gKXOVf joDErG"
  >
    <div
      class="sc-jSMfEi dRbZhr"
    >
      <div
        class="sc-gsnTZi fqhlWJ"
      >
        <div
          class="sc-dkzDqf kUFkcz"
        >
          <h2
            class="sc-bczRLJ hGDuJC"
          >
            Grid two columns
          </h2>
          <p
            class="sc-iBkjds ggWPWg"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Distinctio magnam culpa eveniet doloribus harum? Ipsam, a necessitatibus? Sequi sunt accusantium quod, animi iure a, aliquid dolor ea vel magni dolore?
          </p>
        </div>
        <div
          class="sc-hKMtZM ixKqbH"
        >
          <img
            alt="Grid two columns"
            class="sc-eCYdqJ cukpPu"
            src="assets/images/js.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
